actionview/test/actionpack/controller/render_test.rb

```diff
@@ -37,6 +37,20 @@ def cache_key
   end
 end
 
+module Widgets
+  Table = Struct.new(:title) do
+    include ActiveModel::Conversion
+
+    def to_partial_path
+      "widgets/table"
+    end
+
+    def prefix_partial_path_with_controller_namespace
+      false
+    end
```

Comment:
Looking at the tests you added it seems that what someone would do in their application is add this new method to their model in order to not prefix partials. Is that correct?

This feel like a big long name that's easy to forget and probably meant to be an internal method name. Perhaps the name of this doesn't have to match the internal API so that it's easier for someone to use in their application. Additionally writing a whole method out like this just to set something to false feels overkill.

What about adopting behavior from something like how we do before filters:

```ruby
module Widgets
  prefix_partial_path false
end
```

In Rails we'd then create a public method that defaults to true but can be set to false with the above call.